.storybook/preview-head.html

```diff
@@ -7,4 +7,26 @@
   body {
     background-color: initial !important;
   }
+
+  /**
+   * Percy-specific CSS doesn't play nicely with css-in-js solutions, so we
+   * need to add overrides here instead of in the relevant css files.
+   * See https://docs.percy.io/docs/percy-specific-css
+   *
+   * Selectors need to be in the format `[class^='Thing']` because the actual
+   * class name will have a unique suffix attached to it (e.g. `Thing_20YWn`).
+   *
+   * These overrides exist because normally these values are dynamically
+   * generated in JS and thus the differ between Percy snapshots, which lead
+   * to constant false positives unless we force them to a fixed value.
+  */
```

Comment:
Not sure the first paragraph is that useful?

How about a more minimalist approach:

```js
/**
 * Override dynamically generated values to fixed values instead,
 * eliminating false positives due to ever-changing values between Percy snapshots.
 *
 * Selectors need to be in the format `[class^='Thing']` because compiled
 * class names have unique suffixes attached to them (for example: `Thing_20YWn`).
 * 
 * @see https://docs.percy.io/docs/percy-specific-css
 */
```